GliaWidgets/Sources/Component/Connect/Status/ConnectStatusView.swift

```diff
@@ -2,8 +2,8 @@ import UIKit
 
 final class ConnectStatusView: BaseView {
     private let stackView = UIStackView()
-    private let firstLabel = UILabel()
-    private let secondLabel = UILabel()
+    private let firstLabel = UILabel().makeView()
```

Comment:
What is the difference?

Reply:
This will set `UIView.translatesAutoresizingMaskIntoConstraints` to `false`, enabling autolayout.

Reply:
It is not an issue, but fyi, if view is subview of UIStackView `translatesAutoresizingMaskIntoConstraints` is not necessary.